Open Judge System/Workers/OJS.Workers.Common/Extensions/ProcessExecutionResultExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace OJS.Workers.Common.Extensions
+{
+    using System;
+
+    public static class ProcessExecutionResultExtensions
```

Comment:
Extensions are not necessary here. Simply extract into the the `ProcessExecutionResult` model